### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by ed-phan**
- Create responsive UI
- Create unit tests that test for UI and error display
- Call API, handle success and error routes

### CODE REVIEW

**Review by bao-qnguyen97 — APPROVED** [review]
Awesome, LGTM!!!

File: FE/src/components/LoginForm/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(LoginSchema),
+  });
+
+  const onSubmit = (data) => {
+    axios
+      .post(loginUrl, data)
+      .then((response) => {
+        console.log(response);
```
Comment (Al3xDo) [inline FE/src/components/LoginForm/LoginForm.jsx:28]: I think we should delete it

File: FE/src/components/RegisterForm/RegisterForm.js [hunk 2]
```diff
@@ -1,45 +1,100 @@
...
+    register,
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(RegisterSchema),
+  });
 
-    const onSubmit = data => console.log(data);
+  const onSubmit = (data) => console.log(data);
```
Comment (Al3xDo) [inline FE/src/components/RegisterForm/RegisterForm.js:26]: remove the console.log here

Comment (dungplt242) [reply FE/src/components/RegisterForm/RegisterForm.js:26]: thanks @Al3xDo, this file will be handled in the register-branch.

**Pull request merged**